### PR TITLE
{lang} [foss/2014b] Python-2.7.8-foss-2014b Updating to match intel, (REVIEW)

### PR DESCRIPTION
**Pull request by rjeschmi**
fix url for deap and pull up versions of numpy to the same as intel-2014b

### CODE REVIEW

**Comment by boegel** [comment]
@rjeschmi: needs sync with develop?

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 2 out of 3 (1 easyconfigs in this PR)
Linux centos 6.6, Intel(R) Xeon(R) CPU           E5440  @ 2.83GHz, Python 2.6.6
See https://gist.github.com/18b65fdbff9b174cecb7 for a full test report.

**Comment by boegel** [comment]
@rjeschmi: what's up with the failed test report?

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, Intel(R) Xeon(R) CPU           E5440  @ 2.83GHz, Python 2.6.6
See https://gist.github.com/47798c8c332b64cba84b for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, Intel(R) Xeon(R) CPU           E5440  @ 2.83GHz, Python 2.6.6
See https://gist.github.com/bbd489c672f5d1ed8a25 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/50745c375b858e337559 for a full test report.

**Comment by boegel** [comment]
@rjeschmi: still failing for you?

File: easybuild/easyconfigs/p/Python/Python-2.7.8-foss-2014b.eb [hunk 1]
```diff
@@ -89,9 +94,8 @@ exts_list = [
     ('pyparsing', '2.0.2', {
         'source_urls': ['https://pypi.python.org/packages/source/p/pyparsing/'],
     }),
-    ('pygments', '1.6', {
-        'source_urls': ['https://pypi.python.org/packages/source/P/Pygments/'],
-        'source_tmpl': 'Pygments-%(version)s.tar.gz',
+    ('matplotlib', '1.3.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib/'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.8-foss-2014b.eb]: this shouldn't be added back, it was removed with a reason (extra deps required like libpng)

**Comment by rjeschmi** [comment]
this seems signficant

```

== 2015-06-03 16:00:31,060 main.run DEBUG cmd "export PYTHONPATH=/tmp/eb-wHI6qD/tmpBjiONn/lib/python2.7/site-packages:$PYTHONPATH && python -m timeit -n 3 -r 3 -s "import numpy; x = numpy.random.random((1000, 1000))" "numpy.dot(x, x.T)"" exited with exitcode 132 and output:
/bin/bash: line 1: 13096 Illegal instruction     (core dumped) python -m timeit -n 3 -r 3 -s "import numpy; x = numpy.random.random((1000, 1000))" "numpy.dot(x, x.T)"
 
== 2015-06-03 16:00:31,061 main.run ERROR EasyBuild crashed with an error (at easybuild/tools/run.py:384 in parse_cmd_output): cmd "export PYTHONPATH=/tmp/eb-wHI6qD/tmpBjiONn/lib/python2.7/site-packages:$PYTHONPATH && python -m timeit -n 3 -r 3 -s "import numpy; x = numpy.random.random((1000, 1000))" "numpy.dot(x, x.T)"" exited with exitcode 132 and output:
/bin/bash: line 1: 13096 Illegal instruction     (core dumped) python -m timeit -n 3 -r 3 -s "import numpy; x = numpy.random.random((1000, 1000))" "numpy.dot(x, x.T)"
 

```

**Comment by boegel** [comment]
@rjeschmi: hmm, old OpenBLAS on new system (or other way around)?

**Comment by migueldiascosta** [comment]
@rjeschmi, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**